pyproject.toml

```diff
@@ -1,6 +1,77 @@
+[project]
+name = "pudl_archiver"
+version = "0.2.0"
+authors = [{name = "PUDL", email = "pudl@catalyst.coop"}]
+requires-python = ">=3.10,<3.11"
+
+dependencies = [
+    "catalystcoop.pudl @ git+https://github.com/catalyst-cooperative/pudl.git@dev",
```

Comment:
I believe this (heinous) dependency only exists because we rely on some metadata that is stored in the PUDL repo for the construction of the metadata that's uploaded to Zenodo.  IIRC, getting that split would be both one of the complexities and one of the benefits of pulling the datastore module out of PUDL into a standalone pacakge.

Reply:
Yeah, it seems that way. It's here for now, and one day we'll get rid of it, but this does continue to work. Once we deal with that I think `conda-lock` might even work nicely.